client/ayon_maya/plugins/publish/validate_animation_product_type_publish.py

```diff
@@ -0,0 +1,46 @@
+import ayon_maya.api.action
+from ayon_core.pipeline.publish import (
+    PublishValidationError,
+    ValidateContentsOrder,
+)
+from ayon_maya.api import plugin
+
+
+class ValidateAnimationProductTypePublish(plugin.MayaInstancePlugin):
```

Comment:
This seems like an odd default validation. I can think of plenty reasons you may want to extract both and not just one.